scripts/riscv_bootstrap.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# This script will download and setup the riscv-linux required toolchain and tool.
+
+BOOTSTRAP_SCRIPT_PATH=$(dirname "$0")
+BOOTSTRAP_WORK_DIR=$BOOTSTRAP_SCRIPT_PATH/.bootstrap
+
+function cleanup {
+  if [ -d $BOOTSTRAP_WORK_DIR ]; then
```

Comment:
`[[ … ]]` is preferred over `[ … ]`
Same to other code.

https://google.github.io/styleguide/shellguide.html#s6.3-tests

Reply:
OK